### DISCUSSION TITLE
Could not load file or assembly

### DISCUSSION THREAD

**Discussion by LennieFredriksson** [dotnet/wpf#7228] [Ideas]
Why couldn't there exist one huge cloud distributed bin folder where all projects could point to, so you never ever get the problem of,
"Could not load file or assembly" or "The system cannot find the file specified".
I'm so tired of putting 99% of my time solving that kind of issues and just 1% of developing.

**Comment by miloush** [comment]
I am not sure how this is WPF related or what issues you are facing, but you can in fact implement loading assemblies from a custom place relatively easily, see https://learn.microsoft.com/en-us/dotnet/standard/assembly/resolve-loads
